main/src/Periph/PWM.cpp

```diff
@@ -11,8 +11,7 @@ PWM::PWM(uint8_t pin, ledc_channel_t channel, bool invertDuty) : pin(pin), chann
 			.duty_resolution  = DutyResDefault,
 			.timer_num        = getTimer(channel),
 			.freq_hz          = DefaultFreq,
-			.clk_cfg          = LEDC_AUTO_CLK,
```

Comment:
Jel ovo bio neki warning pa si maknuo? Preferirao bih da ne mjenjamo usput zbog warninga osim ako nam je to eksplicitna namjera da prčkamo po tome.